project3/index.html

```diff
@@ -0,0 +1,409 @@
+ <!DOCTYPE html>
+ <html lang="en">
+
+
+ <head>
+     <meta charset="UTF-8">
+     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+     <link rel="stylesheet" href="style.css">
+     <title>Document</title>
+     <link href="https://fonts.googleapis.com/css?family=Teko&display=swap" rel="stylesheet">
```

Comment:
This doesn't represent all the fonts required for the project or the weights; the correct tag should look like this:
`<link href="https://fonts.googleapis.com/css?family=BenchNine:300,400,700|Teko:300,400,500,600,700&display=swap" rel="stylesheet">`